src/main/java/com/yelp/nrtsearch/server/luceneserver/SearchHandler.java

```diff
@@ -148,7 +147,7 @@ public SearchResponse handle(IndexState indexState, SearchRequest searchRequest)
                 shardState,
                 searchContext.getQueryFields(),
                 grpcFacetResults,
-                threadPoolExecutor,
+                DIRECT_EXECUTOR,
```

Comment:
Can we add some comments here so that it's clear why we are using the direct executor?